pkgs/development/python-modules/click-command-tree/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchpatch
+, fetchFromGitHub
+, click
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "click-command-tree";
+  version = "1.1.0";
+  format = "setuptools";
+  disabled = pythonOlder "3.4";
+
+  src = fetchFromGitHub {
+    owner = "whwright";
+    repo = "click-command-tree";
+    rev = version;
+    sha256 = "sha256-vFOcn+ibyLZnhU3OQMtnHI04UqAY2/CCvhq4EEU4XFo=";
+  };
+
+  patches = [
+    # disable flake8 linting - https://github.com/whwright/click-command-tree/issues/4
+    (fetchpatch {
+      url = "https://github.com/whwright/click-command-tree/commit/1ecfcfa29bf01e1131e6ec712bd7338ac1283dc8.patch";
+      sha256 = "sha256-u5jsNfEo1+XNlkVGPCM/rsDPnYko6cr2z2si9nq+sLA=";
+    })
+  ];
+
+  propagatedBuildInputs = [
+    click
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  pytestFlagsArray = [ "tests.py" ];
+
+  pythonImportsCheck = [
+    "click_command_tree"
+  ];
+
+  meta = with lib; {
+    description = "click plugin to show the command tree of your CLI";
```

Comment:
c -> C

Reply:
This was added to nixpkgs in the meantime in #204536. Should I fix just the typo in this PR or should we wait for the next  update of `click-command-tree` and fix the typo in that PR?

Reply:
Not a big deal if it's already been approved and merged.